File: MegaHerdt.FrontEnd/MegaHerdt/src/app/models/PurchaseClaim/PurchaseClaimAnswer.ts
export class PurchaseClaimAnswer{
    id: number;
    purchaseClaimId: number;
    answer: string;
    employeeUserName: string;

    constructor(){
        this.id = 0;
        this.purchaseClaimId = 0;
        this.answer = '';
        this.employeeUserName = '';
    }
}

File: MegaHerdt.Models/Models/PurchaseClaimAnswer.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MegaHerdt.Models.Models
{
    public class PurchaseClaimAnswer
    {
        [Key]
        public int Id { get; set; }
        [ForeignKey("PurchaseClaim")]
        public int PurchaseClaimId { get; set; }
        public string Answer { get; set; }
        public string EmployeeUserName { get; set; }
        public PurchaseClaim PurchaseClaim { get; set; }
    }
}
